src-tauri/Cargo.toml

```diff
@@ -1,12 +1,15 @@
 [package]
 name = "wooting-macros"
-version = "1.0.2"
+version = "1.1.0-pre-release"
+edition = "2021"
 description = "Macro software made by Wooting."
-authors = ["Jeroen N", "Erik S", "Simon W", "David M", "Khang N"]
+authors = ["Jeroen N", "Erik S", "Simon W", "David M <1900179+medzernik@users.noreply.github.com>", "Khang N", "Pola M"]
 license = "GPL-3.0-only"
 repository = "https://github.com/WootingKb/wooting-macros"
-edition = "2021"
-
+readme = "../README.md"
+homepage = "https://wooting.io/wootomation"
+keywords = ["input", "macros", "wooting"]
+publish = false
```

Comment:
To prevent accidental pushes to a Rust package repo, this is recommended to add.